tsconfig.json

```diff
@@ -1,13 +1,29 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2015",
```

Comment:
Should we override this in the `client` config to be `es5`? Maybe for browser compatibility?

Reply:
Yeah I think it doesn't hurt. Most browsers should be supporting es6 by now but IE still lives

Reply:
I mean doesn't hurt to override it in the client! hurts a little to not support IE

Reply:
I updated the base target to `es5` here 003a11077fdf545ffdc92455ac7df9fdfb3ef6c1 so that all our packages transpile to the same version of js